geopatterns_demo/methods_list.py

```diff
@@ -0,0 +1,13 @@
+from geopatterns_demo.models import Method
+
+def list_creator():
+    """Generate list of the methods geopatterns supports."""
+    methods_list = []
+    for name, label in Method.__members__.items():
+        methods_list.append(
+            {
+                'name': name,
+                'label': label
```

Comment:
While you are at your environment, please try running:

```bash
make lint
```

This will run the linters, they will complain about a few things. Need to satisfy them. In this occasion, they will likely complain on missing comma, which should be like this:

```suggestion
                'label': label,
```